content/vendor/guides/ha-guide.md

```diff
@@ -0,0 +1,411 @@
+    nodeUnreachableToleration: 1m
+```
+
+Then make sure to promote the installer to same channel(s) that you are promoting your app.
+
+## Part I - Setting Up the Cluster
+
+To stand up the cluster, we will be following the these steps:
+
+- [Procuring the (virtual) hardware](#provisioning-the-virtual-hardware)
+- [Setting up and configuring a Load Balancer](#setting-up--configuring-a-load-balancer)
+- [Run the Kots installer on the first node and deploy the application](#run-the-kots-installer--deploy-application)
+- [Verify the Deployment](#verify-the-deployment)
+- [Add Remaining Nodes to the Cluster](#adding-remaining-nodes-to-cluster)
+- [Verify Pods Are Running on All Nodes](#verify-pods-are-running-on-all-nodes)
+
+### Provisioning the (Virtual) Hardware
+
+For this exercise, we are going to create 4 Virtual Machines. One of the VMs will be used to install, configure and run HA Proxy. The remaining 3 VMs will be the 3 Master Nodes for our cluster. 
+
+### Provisioning the HA Proxy VM
+
+For the VM that will host HA Proxy, a bare bones VM should suffice as we will only use it to route traffic. As an example, the command below creates a VM instance with enough juice to run HA Proxy, at least for testing purposes:
+
+```shell
+gcloud compute instances create app-direct-ha-proxy  --boot-disk-size=10GB --labels=app=app-direct --image-project ubuntu-os-cloud --image-family ubuntu-1804-lts --machine-type n1-standard-1
+```
+
+In the command output, you should note the public and internal IP address. The public address of this VM is what we'll use to connect to both the Kots Admin Application console, the application's UI and pgAdmin. We'll use the interal IP address when we install KOTS in HA mode.
+
+If you'd like to explore HA Proxy requirements for heavier workloads, please check HA Proxy's [documentation](https://www.haproxy.com/documentation/hapee/2-0r1/installation/getting-started/os-hardware/)
+
+
+### Provisioning the cluster VMs
+
+Like with the HA Proy VM, keep the [kots System Requirements](https://kurl.sh/docs/install-with-kurl/system-requirements) in mind when provisioning the VMs that will form the cluster. The command below creates a VM that meets those requirements, at least for testing purposes.
+
+```shell
+gcloud compute instances create app-direct-node-01 --boot-disk-size=200GB --labels=app=app-direct --image-project ubuntu-os-cloud --image-family ubuntu-1804-lts --machine-type n1-standard-4
+```
+To create the remaining nodes, simply run the same command but increment the node number (i.e., "app-direct-node-02").
+
+Also note the VMs public and private ip addresses as each VM is provisioned. We'll use the internal ip addresses in the HA Proxy to configure traffic accordingly.
+
+### Setting Up & Configuring a Load Balancer
+
+The main purpose of the load balancer is to direct traffic to the proper node. All end-user interactions (i.e., access the web UI of the deployed application) should go through the load balancer.
```

Comment:
Is "proper" the right word here? Aren't all the nodes technically able to receive traffic? This seems to imply that the load balancer captures logic about what services are running where.

Reply:
I've replaced this with "The main purpose of the load balancer is to give us a single point of access." I think that is really what the load balancer gives us.